File: Unidad 3/JAVA/P7_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p7_u3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class P7_U3 {

    public static void main(String[] args) {
            Scanner sc= new Scanner(System.in);
            double T;
            System.out.println("¿Qué temperatura hay hoy?");
            T = sc.nextInt();

            if (T >= 27)
            {
                System.out.println("¡Hace calor!, ¿vamos por un helado?");
            }
            else if (T < 27 && T >= 20)
            {
                System.out.println("¡El clima hoy es agradable!");
            }
            else
            {
                System.out.println("!Hace frío! deberiamos de abrigarnos.");

            }
        }
    }

File: Unidad 3/C#/P7_U3.cs
using System;

namespace P7_U3 // Note: actual namespace depends on the project name.
{
    internal class Temperatura
    {
        static void Main(string[] args)
        {
            double T;
            Console.WriteLine("¿Qué temperatura hay hoy?");
            T = double.Parse(Console.ReadLine());

            if (T >= 27)
            {
                Console.WriteLine("¡Hace calor!, ¿vamos por un helado?");
            }
            else if (T < 27 && T >= 20)
            {
                Console.WriteLine("¡El clima hoy es agradable!");
            }
            else
            {
                Console.WriteLine("!Hace frío! deberiamos de abrigarnos.");

            }
        }
    }
}